changelog/2.0.0_2022-06-22/update-web-5.7.0-rc.1-beta.4.md

```diff
@@ -22,9 +22,11 @@ We updated ownCloud Web to v5.6.0. Please refer to the changelog (linked) for de
 * Bugfix [owncloud/web#6846](https://github.com/owncloud/web/pull/6846): Upload meta data serialization
 * Bugfix [owncloud/web#7100](https://github.com/owncloud/web/pull/7100): Complete-state of the upload overlay
 * Bugfix [owncloud/web#7104](https://github.com/owncloud/web/pull/7104): Parent folder name on public links
+* Bugfix [owncloud/web#7173](https://github.com/owncloud/web/pull/7173): Re-introduce dynamic app name in document title
+* Bugfix [owncloud/web#7166](https://github.com/owncloud/web/pull/7166): External apps fixes
 
 https://github.com/owncloud/ocis/pull/4005
```

Comment:
Do you want to link your PR here?

Reply:
@micbar please add my PR link to the changelog item when you branch off with the new release branch. CI is green, I want to merge. ;-)